Starcounter.Uniform/ViewModels/UniDataTable.json.cs

```diff
@@ -70,6 +73,18 @@ public void LoadRows()
             this.TotalRows = this.DataProvider.TotalRows;
         }
 
+        public void RemoveRow(Json rowToDelete)
+        {
+            // Check if the data object was deleted before removing the row.
+            // User has to delete data object first so that row can be removed correctly.
+            if(Db.FromId(rowToDelete.Data.GetObjectNo()) != null)
```

Comment:
Isn't it sufficient to check `if(rowToDelete.Data != null)`?

Reply:
Unfortunately not, I checked it, and even after deleting, a reference to the data object is still not nulled.

Reply:
This sounds like a bug in XSON. @miyconst WDYT?